src/biome/text/helpers.py

```diff
@@ -88,3 +91,12 @@ def split_signature_params_by_predicate(
     non_matches_group = list(filter(lambda p: not predicate(p), parameters))
 
     return matches_group, non_matches_group
+
+
+def clean_metric_name(name):
+    if name is not None:
+        new_name = _INVALID_TAG_CHARACTERS.sub("_", name)
+        new_name = new_name.lstrip("/")
+        if new_name != name:
```

Comment:
this statement has no effect to final result

Reply:
done